src/Happyr/LinkedIn/Storage/SessionStorage.php

```diff
@@ -19,7 +19,11 @@ class SessionStorage implements DataStorageInterface
     public function __construct()
     {
         //start the session if it not already been started
-        @session_start();
+        if(php_sapi_name() != "cli") {
```

Comment:
Okey. This lib is much inspired by the Facebook SDK. Sadly that lib is not very good written. 

Thank you. =)